### PR TITLE
fix: prevent focus of disabled select components

### PR DESCRIPTION
**Pull request by jwertkin**

### CODE REVIEW

File: packages/pelagos/src/components/Select.less
Comment (jwertkin) [inline packages/pelagos/src/components/Select.less:46]: This was needed to prevent the `&:hover` background colour from being used when the select is disabled

File: packages/pelagos/__tests__/components/Select-test.js
Comment (jcestibariz) [inline packages/pelagos/__tests__/components/Select-test.js:699]: I'm not using this style anymore, please use `expect(event.preventDefault.mock.calls).toEqual([[]]);

**Review by jcestibariz — CHANGES_REQUESTED** [review]
I'll wait for Matthew's change to be merged before merging this one because I'm not sure what happens with forks.

**Pull request merged**
